src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -346,6 +347,15 @@ private void readFromNBT(@NotNull final NBTTagCompound compound)
             final BlockPos block = BlockPosUtil.readFromNBT(blockTag, TAG_FREE_POSITIONS);
             freePositions.add(block);
         }
+
+        if(compound.hasKey(TAG_HAPPINESS))
+        {
+            this.overallHappiness = compound.getDouble(TAG_HAPPINESS);
+        }
+        else
+        {
+            this.overallHappiness = 5;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Assign this magic number 5 to a well-named constant, and use the constant instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)